packages/atomic/src/components/atomic-result/atomic-result.tsx

```diff
@@ -110,12 +100,6 @@ export class AtomicResult {
       this.getDensityClass(),
       this.getImageClass(),
     ];
-    if (this.isFirstResult()) {
-      classes.push('first');
-    }
-    if (this.isLastResult()) {
-      classes.push('last');
-    }
```

Comment:
This was used to display a divider above every result except the first. It's no longer necessary, and IMO should be handled in the list rather than individual results.

Reply:
Makes sense 👍